From 6411204f0c2a667d93f0987b36373d1dc85474b8 Mon Sep 17 00:00:00 2001
From: Udit Kumar <u-kumar1@ti.com>
Date: Tue, 20 Feb 2024 12:04:38 +0530
Subject: [PATCH 03/27] arm64: dts: ti: k3-j784s4: Update ospi layout

Update size of tiboot3.bin is exceeding allocated
area of 512KB, So assigning 1MB for tiboot3.bin
and adjusting all other offsets.

Cc: Neha Francis <n-francis@ti.com>
Cc: Vaishnav Achath <vaishnav.a@ti.com>
Cc: Apurva Nandan <a-nandan@ti.com>
Signed-off-by: Udit Kumar <u-kumar1@ti.com>
Reviewed-by: Apurva Nandan <a-nandan@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j784s4-evm.dts | 54 ++++++++++++------------
 1 file changed, 27 insertions(+), 27 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-j784s4-evm.dts b/arch/arm64/boot/dts/ti/k3-j784s4-evm.dts
index 8b33751dd8c0..672e6b4d856e 100644
--- a/arch/arm64/boot/dts/ti/k3-j784s4-evm.dts
+++ b/arch/arm64/boot/dts/ti/k3-j784s4-evm.dts
@@ -733,27 +733,27 @@ partitions {
 
 			partition@0 {
 				label = "ospi.tiboot3";
-				reg = <0x0 0x80000>;
+				reg = <0x0 0x100000>;
 			};
 
-			partition@80000 {
+			partition@100000 {
 				label = "ospi.tispl";
-				reg = <0x80000 0x200000>;
+				reg = <0x100000 0x200000>;
 			};
 
-			partition@280000 {
+			partition@300000 {
 				label = "ospi.u-boot";
-				reg = <0x280000 0x400000>;
+				reg = <0x300000 0x400000>;
 			};
 
-			partition@680000 {
+			partition@700000 {
 				label = "ospi.env";
-				reg = <0x680000 0x40000>;
+				reg = <0x700000 0x40000>;
 			};
 
-			partition@6c0000 {
+			partition@740000 {
 				label = "ospi.env.backup";
-				reg = <0x6c0000 0x40000>;
+				reg = <0x740000 0x40000>;
 			};
 
 			partition@800000 {
@@ -790,27 +790,27 @@ partitions {
 
 			partition@0 {
 				label = "ospi_nand.tiboot3";
-				reg = <0x0 0x80000>;
+				reg = <0x0 0x100000>;
 			};
 
-			partition@80000 {
+			partition@100000 {
 				label = "ospi_nand.tispl";
-				reg = <0x80000 0x200000>;
+				reg = <0x100000 0x200000>;
 			};
 
-			partition@280000 {
+			partition@300000 {
 				label = "ospi_nand.u-boot";
-				reg = <0x280000 0x400000>;
+				reg = <0x300000 0x400000>;
 			};
 
-			partition@680000 {
+			partition@700000 {
 				label = "ospi_nand.env";
-				reg = <0x680000 0x40000>;
+				reg = <0x700000 0x40000>;
 			};
 
-			partition@6c0000 {
+			partition@740000 {
 				label = "ospi_nand.env.backup";
-				reg = <0x6c0000 0x40000>;
+				reg = <0x740000 0x40000>;
 			};
 
 			partition@2000000 {
@@ -850,27 +850,27 @@ partitions {
 
 			partition@0 {
 				label = "qspi.tiboot3";
-				reg = <0x0 0x80000>;
+				reg = <0x0 0x100000>;
 			};
 
-			partition@80000 {
+			partition@100000 {
 				label = "qspi.tispl";
-				reg = <0x80000 0x200000>;
+				reg = <0x100000 0x200000>;
 			};
 
-			partition@280000 {
+			partition@300000 {
 				label = "qspi.u-boot";
-				reg = <0x280000 0x400000>;
+				reg = <0x300000 0x400000>;
 			};
 
-			partition@680000 {
+			partition@700000 {
 				label = "qspi.env";
-				reg = <0x680000 0x40000>;
+				reg = <0x700000 0x40000>;
 			};
 
-			partition@6c0000 {
+			partition@740000 {
 				label = "qspi.env.backup";
-				reg = <0x6c0000 0x40000>;
+				reg = <0x740000 0x40000>;
 			};
 
 			partition@800000 {
-- 
2.39.2

